### ISSUE TITLE
RealmWeaver references out of date version of Mono.Cecil

### ISSUE THREAD

**Issue by dansiegel** [realm/realm-dotnet#1364]
## Goals

Compile a project using Realm in Visual Studio for Mac or Mac with the Mono 5.X runtime

## Expected Results

I expect it to compile...

## Actual Results

Error: Fody: The weaver assembly 'RealmWeaver.Fody, Version=1.2.0.0, Culture=neutral, PublicKeyToken=null' references an out of date version of Mono.Cecil.dll (cecilReference.Version). At least version 0.10 is expected. The weaver needs to add a NuGet reference to FodyCecil version 2.0.0.0. (ILWeaving)

* Note that the FodyCecil package does not install in PCL or in iOS/Android projects.

## Steps to Reproduce

Create a new project, and add the Realm NuGet package. Update Fody to v2.0. Attempt to Build the solution.
 -Or-
In a project with Realm already installed, Update Fody to v2.0. Attempt to Build.

## Code Sample

No code needed just add the packages

## Version of Realm and Tooling

NuGet v1.2

Note this issue replaces issue #1342 as Fody has update the package and the Fody out of Memory error no longer applies. This is a new issue that is tied to the Realm NuGet.

**Comment by renzska** [comment]
I've found an issue with Fody 2.0.2 that breaks the process in some cases. I've submitted a PR to Fody that should fix the issue. I'd recommend waiting on Fody to implement the PR and updating to that version in this PR before moving forward: Fody/Fody#306

**Comment by renzska** [comment]
They released Fody 2.0.3 that fixed the issue I mentioned:

https://github.com/Fody/Fody/pull/306
https://github.com/Fody/Fody/issues/305

**Comment by dansiegel** [comment]
Updated to Fody 2.0.3

```
Error: Fody: The weaver assembly 'RealmWeaver.Fody, Version=1.2.0.0, Culture=neutral, PublicKeyToken=null' references an out of date version of Mono.Cecil.dll (cecilReference.Version). At least version 0.10 is expected. The weaver needs to add a NuGet reference to FodyCecil version 2.0.3.0. (RealmTest)
```

**Comment by HobDev** [comment]
updated to Fody 2.0.5

`Severity	Code	Description	Project	File	Line	Suppression State
Error		Fody: The weaver assembly 'RealmWeaver.Fody, Version=1.2.0.0, Culture=neutral, PublicKeyToken=null' references an out of date version of Mono.Cecil.dll (cecilReference.Version). At least version 0.10 is expected. The weaver needs to add a NuGet reference to FodyCecil version 2.0.	`

**Comment by nirinchev** [comment]
We're working on updating to latest Fody and hope to have something releasable in the near future.

**Comment by dansiegel** [comment]
@nirinchev tested this with 1.2.1, and everything works as expected. Thanks!
